--- mochad-0.1.17/global.h	2022-08-24 19:21:16.906783768 -0300
+++ mochad-0.1.17patched/global.h	2022-08-24 19:21:17.894756328 -0300
@@ -22,14 +22,13 @@
     const char *name;
 };
 
-int Cm19a;
-int PollTimeOut;
+extern int Cm19a;
+extern int PollTimeOut;
 
 /* 1 bit per house code, 1=RF to PL, 0=off, default all house codes on */
-unsigned short RfToPl16;
-
-unsigned short RfToRf16;
+extern unsigned short RfToPl16;
 
+extern unsigned short RfToRf16;
 
 #define dbprintf(fmt, ...) _dbprintf(fmt, __FILE__,__LINE__, ## __VA_ARGS__)
 int _dbprintf(const char *fmt, ...);
@@ -48,5 +47,3 @@
 int or20client(int fd);
 
 int del_client(int fd);
-
-
